frontend/tet/admin/src/components/editor/companyInfo/CompanyInfo.tsx

```diff
@@ -25,15 +24,19 @@ const CompanyInfo: React.FC = () => {
 
   const keywordsResults = useQuery(['keywords', addressSearch], () => getAddressList(addressSearch));
 
-  const keywords: OptionType[] = keywordsResults.data
+  const keywords: LocationType[] = keywordsResults.data
```

Comment:
map methods can be heavy to calculate every render so I would suggest to wrap it into `React.useMemo` -hook